packages/botonic-plugin-contentful/src/manage-cms/manage-cms-error.ts

```diff
@@ -26,6 +26,20 @@ export class ErrorReportingManageCms implements ManageCms {
       .catch(this.handleError('updateFields', context, contentId, fields))
   }
 
+  deleteEntry<T extends cms.Content>(contentId: ContentId): Promise<void> {
+    return this.manageCms
+      .deleteEntry(contentId)
+      .catch(this.handleError('deleteEntry', undefined, contentId, {}))
+  }
+
+  createEntryWithId<T extends cms.Content>(
```

Comment:
I think that often we will need to create entries with autogenerated id. To avoid having 2 different calls, I suggest changin to :
`createEntry(readonly model: ContentType, id: string): Promise<ContentId>`

and in the future if we want to support autogenerate id, we add "?" to id and call contentful createEntryWithId (without breaking compatibility)